nbs/tsdataset.ipynb

```diff
@@ -54,13 +54,15 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.            if isinstance(idx, int):
</pre><p>Can you please switch the order here to avoid indenting such a large portion? I realize that's the way the other method is currently defined but it's easier to read like:</p>

<pre class="ql-syntax" spellcheck="false">if not isinstance(idx, int):
    raise ValueError(f'idx must be int, got {type(idx)}')
# actual logic
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>